<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bankChoice</class>
 <widget class="QWidget" name="bankChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="BankList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <underline>true</underline>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ChooseBank">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Вибрати Банк</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateBank">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>70</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Створити новий банк</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Видалити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
